Gemfile

```diff
@@ -55,16 +50,21 @@ gem 'validates_email_format_of'
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platform: :mri
+  gem 'codecov', require: false
   gem 'dotenv-rails'
   gem 'equivalent-xml', '~> 0.6.0'
   gem 'guard-rspec'
   gem 'guard-rubocop'
   gem 'rspec'
   gem 'rspec-rails', '~> 3.4'
+  gem 'rspec_api_documentation', require: false
```

Comment:
We're going to test this in our build system. I think these specific gems were in the production group because our Docker build runs the `docs:generate`. I'll let you know if we need to move these back.
https://github.com/CruGlobal/mobile-content-api/blob/master/Dockerfile#L19

Reply:
I see, thanks for clarifying (it seemed confusing) 
but since that command sets `RAILS_ENV=test` it should do fine
I am probably just being over-cautions here as I've been bitten by rspec previously ... 
and it makes me worried that by any chance it might get loaded on production

Reply:
restored the implicit require: https://github.com/CruGlobal/mobile-content-api/pull/181/commits/927c8dfaab0b985ef2889eb59cdefd3d9e6144c6

and now `bundle exec rake db:create db:setup docs:generate RAILS_ENV=test` works

thanks for pointing this one out